pywinauto/unittests/test_common_controls.py

```diff
@@ -336,6 +336,19 @@ def testItemClickInput(self):
         self.assertEquals(self.ctrl.GetItem('Green').IsFocused(), True)
         self.assertEquals(self.ctrl.IsFocused('Green'), True) # TODO: deprecated method
         self.assertEquals(self.ctrl.GetItem('Magenta').IsSelected(), False)
+		
+		# Test click on checkboxes
+        self.dlg.Toolbar.Button(6).Click() # switch on states
+        
+        self.ctrl.GetItem('Red').ClickInput(where='check')

```

Comment:
Great! But `Click(where="check")` is not covered (see [the pull request coverage for common_controls.py](https://codecov.io/github/pywinauto/pywinauto/pywinauto/controls/common_controls.py?ref=ffae667a1473b45a1e9f61b69b78806fdd5b0e6e)). We also need to cover "Full Row Details" case though it's quite experimental. When you customize AppVeyor project, please also sign up to codecov.io. Code coverage info will be submitted automatically for your fork after each AppVeyor build.